language-plutus-core/stdlib/Language/PlutusCore/StdLib/Everything.hs

```diff
@@ -23,6 +26,8 @@ import           Language.PlutusCore.StdLib.Data.Unit
 import           Language.PlutusCore.StdLib.Meta.Data.Tuple
 import           Language.PlutusCore.StdLib.Type
 
+type PLCTerm = Term TyName Name
```

Comment:
I'd prefer to avoid local aliases for common types - if we're going to do this we should do it somewhere centrally and use it consistently in this library.